app/frontend/components/pages/login/loginPage.tsx

```diff
@@ -16,190 +17,198 @@ import Tag from '../../common/tag'
 import WalletLoadingErrorModal from './walletLoadingErrorModal'
 import {getTranslation} from '../../../translations'
 import {errorHasHelp} from '../../../helpers/errorsWithHelp'
-// import {ADALITE_CONFIG} from '../../../config'
+import {AuthMethodEnum} from '../../../state'
 
+// TODO: extract from dashboardPage after rebase
+const useViewport = () => {
+  const [isViewportSmall, setIsViewportSmall] = useState(false)
+
+  const handleScreenResize = () => {
+    setIsViewportSmall(window.innerWidth < 1024)
+  }
+  useEffect(() => {
+    handleScreenResize()
+    window.addEventListener('resize', handleScreenResize)
+
+    return () => window.removeEventListener('resize', handleScreenResize)
+  }, [])
+
+  return isViewportSmall
+}
+
+// TODO: extract to app/frontend/constants.ts after rebase
 const AUTH_METHOD_NAMES = {
```

Comment:
Im not sure if you meant to do this after the rebase as well, but we could make this into an enum and use it in this file a bit more. Most of the components here use the literar strings like `'mnemonic'` and `'hw-wallet'`...
Also it might be nice to check where else we are using the literal values, like in `actions.ts` in functions like `setAuthMethod`,  `loadDemoWallet` and `openGenerateMnemonicDialog`.

Reply:
Yeah it's used with string literals in many places, I stated it as possible future changes, but I can do it now

Reply:
Sorry, I wasnt sure if you just wanted to extract it to `constants.ts` or also do this, but still, If it doesn't bother you that much we should do it now, I can hardly imagine somebody getting back to it later just to fix this.